### PR TITLE
changes to 2nd/3rd paragraphs of intro

### PR DESCRIPTION
**Pull request by davharris**
Some issues this addresses:

* Cites a deep learning review early on
* AlphaGo isn't off-the shelf.  It includes multiple networks, Monte Carlo tree search, etc.
* The Quantum paper seems to be using RBMs, which aren't deep unless you stack them. 
* Introduces the supervised/unsupervised distinction, which is necessary for readers to understand why the cat detector is cool

### CODE REVIEW

**Comment by agitter** [comment]
@bdo311, are you okay removing the off-the-shelf phrase?

**Comment by davharris** [comment]
@agitter it just gets moved to line 20

File: sections/02_intro.md [hunk 1]
```diff
@@ -11,23 +11,31 @@ knowledge that allow us to better treat, categorize, or study disease, all
...
-
-What if, more generally, deep learning could solve the challenges presented by
-the growth of data in biomedicine? Could these algorithms identify the "cats"
-hidden in our data - the patterns unknown to the researcher - and act on them?
-In this review, we examine whether deep learning's transformation of biomedical
-science is simply a matter of time or if there are unique challenges posed by
-biomedical data that render deep learning methods either more challenging or
-less fruitful to apply.
+learning has revolutionized image classification and speech recognition due to
+their flexibility and high accuracy [@doi:10.1038/nature14539]. More recently,
```
Comment (agitter) [inline sections/02_intro.md]: Should this be singular?  "its flexibility" to match "a class"?

File: sections/02_intro.md [hunk 2]
```diff
@@ -11,23 +11,31 @@ knowledge that allow us to better treat, categorize, or study disease, all
...
+these algorithms have shown equally promising results in fields as diverse as
+high-energy physics [@doi:10.1038/ncomms5308], dermatology
+[@doi:10.1038/nature21056], translation among written languages
+[@arxiv:1609.08144], and drug discovery [@doi:10.1002/minf.201501008]. Across
```
Comment (agitter) [inline sections/02_intro.md]: Despite my personal enthusiasm for deep learning in drug discovery, I suggest removing it from this list.  There is fantastic research in this area, but it has not been an obvious success like those other areas.

**Comment by davharris** [comment]
@agitter 👍

**Comment by bdo311** [comment]
looks good to me! thanks!

**Comment by agitter** [comment]
Merging this

**Pull request merged**